### DISCUSSION TITLE
Dependabot Yarn 4 support

### DISCUSSION THREAD

**Discussion by RobinClowers** [community/community#129674] [Code Security]
### Select Topic Area

Product Feedback

### Body

I see in the [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems) that Yarn v3 is supported, but not v4. The latest version of yarn brings a significant performance improvement, it would be really useful if we could get support for it!

**Comment by eli-gc** [comment]
Apparently it is supported according to this anyways: [https://github.com/dependabot/dependabot-core/issues/8610](https://github.com/dependabot/dependabot-core/issues/8610), but I am having trouble with authentication to a private registry.

**Comment by RobinClowers** [comment]
Yes, it shipped a while back, and it's been working well for my team. We don't have any private registries, so I'm afraid I'm not help there. Probably best to file a separate ticket on that.
